numpy/char/__init__.pyi

```diff
@@ -0,0 +1,58 @@
+from numpy.core.defchararray import (
+    equal as equal,
+    not_equal as not_equal,
+    greater_equal as greater_equal,
+    less_equal as less_equal,
+    greater as greater,
+    less as less,
+    str_len as str_len,
+    add as add,
+    multiply as multiply,
+    mod as mod,
+    capitalize as capitalize,
+    center as center,
+    count as count,
+    decode as decode,
+    encode as encode,
+    endswith as endswith,
+    expandtabs as expandtabs,
+    find as find,
+    index as index,
+    isalnum as isalnum,
+    isalpha as isalpha,
+    isdigit as isdigit,
+    islower as islower,
+    isspace as isspace,
+    istitle as istitle,
+    isupper as isupper,
+    join as join,
+    ljust as ljust,
+    lower as lower,
+    lstrip as lstrip,
+    partition as partition,
+    replace as replace,
+    rfind as rfind,
+    rindex as rindex,
+    rjust as rjust,
+    rpartition as rpartition,
+    rsplit as rsplit,
+    rstrip as rstrip,
+    split as split,
+    splitlines as splitlines,
+    startswith as startswith,
+    strip as strip,
+    swapcase as swapcase,
+    title as title,
+    translate as translate,
+    upper as upper,
+    zfill as zfill,
+    isnumeric as isnumeric,
+    isdecimal as isdecimal,
+    array as array,
+    asarray as asarray,
+    compare_chararrays as compare_chararrays,
+    chararray as chararray
+)
+
+__all__: list[str]
+__path__: list[str]
```

Comment:
```suggestion
```
Minor nit: `__path__` is already handled and special cased by type checkers these days.

Reply:
Thank you for pointing that out!